tests/Oryx.Integration.Tests/Php/PhpExifTest.cs

```diff
@@ -26,19 +26,19 @@ public PhpExifTest(ITestOutputHelper output, TestTempDirTestFixture fixture)
         // platform-version in it's own pipeline agent. This is
         // because our agents currently a space limit of 10GB.
         [Fact, Trait("category", "php-8.0")]
-        public void PipelineTestInvocationsPhp80()
+        public async void PipelineTestInvocationsPhp80()
         {   
             string phpVersion80 = "8.0";
-            ExifExample(phpVersion80);
-            PhpFpmExifExample(phpVersion80);
+            await ExifExample(phpVersion80);
+            await PhpFpmExifExample(phpVersion80);
         }
 
         [Fact, Trait("category", "php-7.4")]
-        public void PipelineTestInvocationsPhp74()
+        public async void PipelineTestInvocationsPhp74()
```

Comment:
I didn't look into all the new `await` changes, but is it possible that they were run without `await` on purpose, to run them in parallel? I'm not saying that this was good though, because without `await` we would swallow exceptions rather than failing the test.
Should we consider using a `Task.WaitAll()` or similar to keep the parallelism while respecting async calls?